documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc

```diff
@@ -3,81 +3,88 @@
-| `--package-name` or `-pn` | Set the package name for generated Java files. This applies to `-java` and `-static-java`.
-| `--disable-jackson-annotations` or `-dja` | This option is to be used together with `-java` or
-  `-static-java` and will lead to generated Java code that does not contain
-  https://github.com/FasterXML/jackson[Jackson] Annotations.
-| `--execute-library-macros` or `-elm` | This option is used together with `-java` or `-static-java` and lead to generated Java code which includes the output from the custom macros provided via `-tlp` and `-tlfn`.
-| `--template-library-file` or `-tlf` | The path and name of the template file containing custom macros.
-It is used together with `-elm`.
-| `--use-semantic-api-version` or `-sv` | This option is used together with `-oapi-json` or `-oapi-yaml` in order to use
-  the complete semantic version of the Aspect Model as the version of the Aspect API.
-| `--aspect-api-base-url` or `-base-url` | This option is used together with `-oapi-json` or `-oapi-yaml` to provide the
-  base URL for the Aspect API OpenAPI specification.
-| `--include-aspect-query-api` or `-query-api` | This option is used together with `-oapi-json` or `-oapi-yaml` to include
-  the path for the Query Aspect API Endpoint in the OpenAPI specification.
-| `--aspect-exclude-paging` or `-no-paging` | This option is used to exclude the paging information in the OpenAPI specification. Paging is per default activated.
-| `--aspect-time-based-paging` or `-atbp` | This option is used to set the used paging strategy as time-based paging.
-| `--aspect-offset-based-paging` or `-aobp` | This option is used to set the used paging strategy as offset-based paging.
-| `--aspect-cursor-based-paging` or `-acbp` | This option is used to set the used paging strategy as cursor-based paging.
-| `--aspect-resource-path` or `-arp` | This option is used together with `-oapi-json` or `-oapi-yaml` to set the resource-path`
-  for the Aspect API endpoints.
-| `--aspect-parameter-file` or `-apf` | This option is used together with `-oapi-json` or `-oapi-yaml` to set a file including
-  the schema description for the resource. For JSON the description has to be in json, for YAML it has to be in YAML
+| Command | Description/Options | Examples
+| help                                  | Get overview of all commands                                   | bamm-cli help
+| help <subcommand>                     | Get help for a specific subcommand                             | bamm-cli help aspect
+| aspect help <aspect subcommand>       | Get help for `aspect` subcommands                              | bamm-cli aspect help validate
+| aspect <model> validate               | Validate Aspect Model                                          | bamm-cli aspect AspectModel.ttl validate
+| aspect <model> prettyprint            | Pretty-print Aspect Model                                      | bamm-cli aspect AspectModel.ttl prettyprint
```

Comment:
This table currently looks like this:
![image](https://user-images.githubusercontent.com/1199036/160873136-c5b9b38c-94f0-4db2-b443-3e932a3443b7.png)
I think it would look better and be easier to understand, if the table cells of the top-level commands in the first column span all rows of the options for this command, i.e. like this:
![image](https://user-images.githubusercontent.com/1199036/160873585-debb7075-38d4-41a3-9541-96278eedc7c3.png)
This is described [here](https://docs.asciidoctor.org/asciidoc/latest/tables/span-cells/#span-multiple-rows); you'd add for example `.2+` before the `|` of the first column to have this cell span two rows, and then remove the `|` at the beginning of the next two lines.